docs/.vuepress/atomicDEX-sidebar.js

```diff
@@ -282,95 +269,113 @@ let atomicDEXsidebar = {
-          "trade_preimage",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md",
-          "get_raw_transaction",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/recreate_swap_data.md",
-          "recreate_swap_data",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/stop_simple_market_maker_bot.md",
-          "stop_simple_market_maker_bot",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/start_simple_market_maker_bot.md",
-          "start_simple_market_maker_bot",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/telegram_alerts.md",
-          "telegram_alerts",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/add_delegation.md",
-          "add_delegation",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/remove_delegation.md",
-          "remove_delegation",
-        ],
-        [
-          "/basic-docs/atomicdex-api-20-dev/get_staking_infos.md",
-          "get_staking_infos",
-        ],
-        ["/basic-docs/atomicdex-api-20-dev/withdraw.md", "withdraw"],
+        {
+          title: "Orders",
+          collapsable: true,
+          children: [
+            ["/basic-docs/atomicdex-api-20-dev/best_orders.md", "best_orders"],
+          ]
+        },
+        {
+          title: "Seednode Version Stats",
+          collapsable: true,
+          children: [
+            [
+              "/basic-docs/atomicdex-api-20-dev/add_node_to_version_stat.md",
+              "add_node_to_version_stat",
+            ],
+            [
+              "/basic-docs/atomicdex-api-20-dev/remove_node_from_version_stat.md",
+              "remove_node_from_version_stat",
+            ],
+            [
+              "/basic-docs/atomicdex-api-20-dev/start_version_stat_collection.md",
+              "start_version_stat_collection",
+            ],
+            [
+              "/basic-docs/atomicdex-api-20-dev/stop_version_stat_collection.md",
+              "stop_version_stat_collection",
+            ],
+            [
+              "/basic-docs/atomicdex-api-20-dev/update_version_stat_collection.md",
+              "update_version_stat_collection",
+            ]
+          ]
+        },
+        {
+          title: "Staking",
+          collapsable: true,
+          children: [
+            [
+              "/basic-docs/atomicdex-api-20-dev/add_delegation.md",
+              "add_delegation",
+            ],
+            [
+              "/basic-docs/atomicdex-api-20-dev/get_staking_infos.md",
+              "get_staking_infos",
+            ],
+            [
+              "/basic-docs/atomicdex-api-20-dev/remove_delegation.md",
+              "remove_delegation",
+            ]
+          ]
+        },
+        {
+          title: "Utility",
+          collapsable: true,
+          children: [
+            ["/basic-docs/atomicdex-api-20-dev/recreate_swap_data.md", "recreate_swap_data"],
```

Comment:
I think these can be moved to `Swaps` section.

Reply:
There was no "swaps" subsection in the 2.0 sections, so I renamed "Utility" to "Swaps". In the legacy section, I also moved swaps related methods from utility subsection to swaps subsection.